strings.libsonnet

```diff
@@ -0,0 +1,127 @@
+local d = import 'doc-util/main.libsonnet';
+
+{
+'#': d.pkg(
+    name='strings',
+    url='github.com/jsonnet-libs/xtd/strings.libsonnet',
+    help=|||
+        `strings` implements few helpers functions to manipulate strings.
+    |||
+    ),
+
+    isUpcase(c)::    std.codepoint(c) >= 65 && std.codepoint(c) <= 90,   // between A & Z
+    isDowncase(c)::  std.codepoint(c) >= 97 &&  std.codepoint(c) <= 122, // between a & z
+    isDigit(c) ::    std.codepoint(c) >= 48 && std.codepoint(c) <= 57,   // between 0 & 9
+    isAlphaNum(c)::  self.isUpcase(c) || self.isDowncase(c) || self.isDigit(c),
```

Comment:
These are already covered in the `ascii` package, perhaps we can reference those instead of redefining them here.